pkg/vault/test/Authorizer.test.ts

```diff
@@ -20,6 +20,8 @@ describe('Authorizer', () => {
   const ROLE_2 = '0x0000000000000000000000000000000000000000000000000000000000000002';
 
   const ROLES = [ROLE_1, ROLE_2];
+  const WHERE = [ethers.Wallet.createRandom().address, ethers.Wallet.createRandom().address];
+  const NOT_WHERE = ethers.Wallet.createRandom().address;
```

Comment:
Huh I didn't know about `createRandom()`, that's cool! We often use the `other` signer to represent this, but this is a nifty tool.

Reply:
I was spending too much time context-switching between the test setup files and the test files to really get which address what was, and I think at some point a test was failing because I misused one, so I thought 'there must be a better way'